Automattic/vip-go-node
# Move from package-lock.json to npm-shrinkwrap.json

## Description

To help enforce dependencies since package-lock is not shipped with npm releases.

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm i`.
